### PR TITLE
participant-integration-api: Move `SeedService` here. [KVL-1002]

### PR DESCRIPTION
**Pull request by SamirTalwar-DA**
I don't see a reason why it's part of the participant state API, and it definitely doesn't need to change between v1 and v2.

### Changelog

- **[Integration Kit]** The class `SeedService` has been moved from the _participant-state_ Maven package to the _participant-integration-api_ Maven package, under the Java package name `com.daml.platform.apiserver` to reflect its usage by the API server, not the participant state API. If you use this class directly, you will need to change your imports.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by miklos-da — APPROVED** [review]
Thanks for moving `SeedService` out from `participant-state`!

File: ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala [hunk 1]
```diff
@@ -29,15 +30,15 @@ import com.daml.ledger.api.refinements.ApiTypes.ApplicationId
...
 import com.daml.ledger.api.tls.TlsConfiguration
 import com.daml.ledger.api.v1.{value => v}
-import com.daml.ledger.client.{LedgerClient => DamlLedgerClient}
 import com.daml.ledger.client.configuration.{
   CommandClientConfiguration,
   LedgerClientConfiguration,
   LedgerIdRequirement,
 }
-import com.daml.ledger.participant.state.v1.SeedService.Seeding
+import com.daml.ledger.client.{LedgerClient => DamlLedgerClient}
```
Comment (S11001001) [inline ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala:38]: Please don't use import cleaning in any way. We don't have an idempotent approach.

**Review by S11001001 — APPROVED** [review]

File: ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala (hunk 1 above)
Comment (SamirTalwar-DA) [reply ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala:38]: I know it's not always idempotent, but I did verify that reordering these imports won't affect any behavior.

There are large parts of the codebase where import ordering is standard (e.g. pretty much everything under _ledger_), so I'd rather not turn it off. If you like, I can revert this file.

**Review by fabiotudone-da — APPROVED** [review]

**Comment by SamirTalwar-DA** [comment]
Gonna merge in the interest of moving on, but please feel free to comment regardless and I'll address changes later.

File: ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala (hunk 1 above)
Comment (S11001001) [reply ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala:38]: > I did verify that reordering these imports won't affect any behavior.

I don't think this is a good reason to add such hunks. I can do lots of import reordering that doesn't affect their behavior; even so, I'm still gumming up the review (and potentially forcing conflicts) with totally irrelevant diff lines.

> There are large parts of the codebase where import ordering is standard

I would say that it is more the case that they _look_ standard by the eye test, but do not actually match up to any specific standard. You will never reach an equilibrium with this approach, as things stand, but instead will add diff/conflict churn multiplied by however many people also do import reordering.

Comment (SamirTalwar-DA) [reply ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala:38]: That's fair. I agree that it's kind of a mess and not everyone follows the same conventions, uses the same version of the IDE, remembers to run the import reorganizer, or is in a situation where there is no equilibrium.

BRB gonna go write a tool to do this in a "standard" way.

Comment (S11001001) [reply ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala:38]: Alright, see you in a couple years 🤣

**Pull request merged**